### ISSUE TITLE
LVGL Flow: Add event handling for pages/screens

### ISSUE THREAD

**Issue by fietser28** [eez-open/studio#316]
LVGL has several events specific to screens (load/unload). These events can currently not be used in flow because there is no event handling and interface for it.

It is convenient to have this interface available to trigger actions/flows when a screen is loaded (displayed) or when left.

In the UI this probably means:
- the properties tab of a screen get an events section similar to other widgets.
- If flow events are defined in the properties handles appear at the screen borders, similar to other widgets.

This issue might also prepare for partial implementation of #277?

**Comment by mvladic** [comment]
Done.

**Comment by fietser28** [comment]
2 observations:
- It is possible to add screen_(un)load(ed) events to widgets (like buttons/panels). In my interpretation of the LVGL documentation this is fine but probably not intentional. It becomes a problem:
- When I create a screen flow event in a button and later a screen event for the screen then making the connection in the editor connects to the wrong handle:

<img width="1078" alt="image" src="https://github.com/eez-open/studio/assets/8796121/508a823f-3574-4d0d-9ecb-00a211932956">
In this example I pressed on the handle of the screen but the line to the button appears (the mouse crosshair is not in the screenshot but at the floating line). The button handle is the event shown in the properties.

Removing the event on the button allows me to connect to the screen.

I don't see a real use case for connecting handles to widgets, so this probably should be disabled. 
(I made a mistake in a different project and had to find the culprit in the eel-project file manually)

Apart from this, it works in the simulator. Will also test this on real hardware.
